### ISSUE TITLE
Publish a debug image with reach stable/latest release.

### ISSUE THREAD

**Issue by achrefbensaad** [kubearmor/KubeArmor#1059]
## Feature Request

**Short Description**

We want to have a debuggable release (pprof enabled) along side the nomal release.

**Describe the solution you'd like**

We would like to enable pprof conditionally when compiling KubeArmor binary.
e.g:  `ENABLE_PPROF=true make run` to enable pprof and run KubeArmor.

We also would like to push for each release(stable or latest) a second image with pprof enabled.
The image should be named as follow: `<version/state>-debug`
eg: `latest-debug` `stable-debug` `v0.8-debug`

**Comment by mdsahiloss** [comment]
@achrefbensaad I would like to work on this.

**Comment by achrefbensaad** [comment]
Sure.

**Comment by mdsahiloss** [comment]
@achrefbensaad is there kubeArmor using `pprof`? I don't see any usage of `pprof`in this repository. If so, Please show me the path so that I can proceed further with this issue.

**Comment by achrefbensaad** [comment]
Hi @MdSahil-oss , we dont have pprof implemented in this repository. We would like to implement it.

**Comment by mdsahiloss** [comment]
Thanks @achrefbensaad for the confirmation, I would like to implement it but first let me know, How would it be implemented as a server or something else.

**Comment by mdsahiloss** [comment]
@achrefbensaad As I can see previously `pprof` was available in this repo as web server by the [commit](https://github.com/kubearmor/KubeArmor/commit/5203c28eef2abff58ea2474816815b3fce37f505). if `pprof` has to be added as server then I can take refrence of this [commit](https://github.com/kubearmor/KubeArmor/commit/5203c28eef2abff58ea2474816815b3fce37f505)

**Comment by achrefbensaad** [comment]
@MdSahil-oss , yes we can use that commit as a base and extend it to expose the following functions on these API's

- pprof.Index -> /debug/pprof
- pprof.Cmdline -> /debug/pprof/cmdline
- pprof.Profile -> /debug/pprof/profile
- pprof.Symbol -> /debug/pprof/symbol
- pprof.Handler("heap") -> /debug/pprof/heap
- pprof.Handler("block") -> /debug/pprof/block
- pprof.Handler("goroutine") -> /debug/pprof/goroutine
- pprof.Handler("threadcreate") ->/debug/pprof/threadcreate
